src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -221,6 +231,30 @@ export const CustomerInstitutionMetadataForm = ({
               </Field>
             </div>
 
+            <Divider />
+
+            <Field name={CustomerInstitutionFieldNames.InactiveFrom}>
+              {({ field }: FieldProps<string | null | undefined>) => (
+                <FormControlLabel
+                  label={t('basic_data.institutions.institution_is_inactive')}
+                  control={
+                    <Checkbox
+                      onChange={(_event, checked) => {
+                        setFieldValue(
+                          CustomerInstitutionFieldNames.InactiveFrom,
+                          checked ? new Date(Date.now()).toISOString() : null
+                        );
+                      }}
+                      data-testid={dataTestId.basicData.institutionAdmin.inactiveCheckbox}
+                      checked={isInactive(field.value)}
```

Comment:
Siden dato alltid er litt krøkkete (ref [denne](https://github.com/BIBSYSDEV/NVA-Frontend/pull/5083/files#r1471562700) kommentaren), så er det kanskje like greit å håndtere heltet som en bool for klienten sin del? Forstår det som at det i praksis blir det samme 
```suggestion
                      checked={!!field.value}
```

Reply:
Ja, lurte på om jeg skulle være for nøye. 
Backenden vil vurdere datoen slik at hvis datoen er satt i fremtiden, så er institusjonen fortsatt aktiv og brukere får sine aksessrettigheter for institusjonen når de logger på. 

Med dato-sjekken i klienten vil visninga av checkboxen være i sync med backenden, men siden det ikke finnes funksjonalitet for brukeren for å sette datoen inn i framtida, så er det jo litt teoretisk.

Reply:
Fortsetter tråden over, her, siden det egentlig er her den er relevant.

Jeg tenker på at brukeren har annet (feil) oppsett på sin maskin ja. Dato er alltid kødd, feks bytter norge mellom å være UTC+1 (vintertid) og UTC+2 (sommertid), så det er 2 timer i året hvor ting er bingo, og i dette tilfellet vil det bli feil om man gjør dette rundt overgang fra sommertid til vintertid.

Side note: Vi har utfordringer med dette flere plasser fra før, siden klienten setter midnatt som tidspunkt når man skal velge dato. Feks om noen setter en publiseringsdato til 10.05.24, så settes det til midnatt i den tidssonen (UTC+2 om det blir gjort om sommeren). Om man åpner den posten i vintertid (UTC+1) vil det trekkes fra en time, sånn at det blir kl 23 dagen før, og frontend vil vise 09.05.24 🤦.

Har en antagelse om at hvis det er en verdi her så skal den alltid være markert som inaktiv. Men dette er ikke kjempeviktig, så lever greit med det sånn det er også 👍

Reply:
Setter den til å ta se om verdien er null eller ikke. Så får vi ta problemet hvis det kommer at noen har lyst til å inaktivere noe automatisk i morgen.